app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] According to the [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), associations between models are set up correctly, something like so: 

```ruby
belongs_to :author, class_name: 'User'
belongs_to :post
```
Please kindly fix that.

Reply:
@t-yanick  thanks I fixed it.